plugins/resources/replication/irods_create_write_replicator.cpp

```diff
@@ -9,10 +9,10 @@ namespace irods {
     create_write_replicator::create_write_replicator(
         const std::string& _root_resource,
         const std::string& _current_resource,
-        const std::string& _child ) {
-        root_resource_ = _root_resource;
-        current_resource_ = _current_resource;
-        child_ = _child;
+        const std::string& _child )
+      : root_resource_(_root_resource),
+        current_resource_(_current_resource),
+        child_(_child){
```

Comment:
We have generally been using list-(braced-?)initialization in our constructors by convention. Any reason in particular to switch to parentheses? I've seen some strong opinions around "universal initialization" so I think we are willing to listen to a different view on things here. Same comment for all the other spots this is happening

Reply:
I hadn't honestly thought to use it while writing the constructor